Dockerfile

```diff
@@ -62,6 +62,11 @@ COPY ci/operator-keypairs/id.json /root/.config/solana/id.json
 COPY ci/operator-keypairs/id2.json /root/.config/solana/id2.json
 COPY ci/keys/ /opt/keys
 
+ARG ROCKSDB_HOST
+ARG ROCKSDB_PORT
+
 ENV PATH=${PATH}:/opt
+ENV ROCKSDB_HOST=${ROCKSDB_HOST}
+ENV ROCKSDB_PORT=${ROCKSDB_PORT}
```

Comment:
Don't add environment variables this way. You always can add it inside `environment` section in docker-compose file